packages/ethers/src/index.ts

```diff
@@ -130,18 +135,17 @@ export class TurnkeySigner extends ethers.Signer implements TypedDataSigner {
     // https://github.com/ethers-io/ethers.js/blob/f97b92bbb1bde22fcc44100af78d7f31602863ab/packages/wallet/src.ts/index.ts#L117-L121
     if (unsignedTx.from != null) {
       const selfAddress = await this.getAddress();
-      if (ethers.utils.getAddress(unsignedTx.from) !== selfAddress) {
+      if (ethers.getAddress(unsignedTx.from.toString()) !== selfAddress) {
         throw new Error(
           `Transaction \`tx.from\` address mismatch. Self address: ${selfAddress}; \`tx.from\` address: ${unsignedTx.from}`
         );
       }
 
-      delete unsignedTx.from;
+      // maybe we no longer need to drop this
+      // delete unsignedTx.from;
```

Comment:
Can you explain your reasoning? Generally, when it doubt: ✂️ -- we can always add this back later if needed.

Reply:
I believe ethers v6 now has native support for eip1193 (no `Eip1193Bridge` required). However, I'm not sure this is currently unit testable in this way (node environment). I've reached out to the Ethers team to see if there is a new, Ethers v6-specific way of creating an eip1193 provider.

If they don't come up with a solution in a timely manner or say it's currently not supported, I'm also down to remove this test for now and revisit this topic altogether once we upgrade demo-consumer-wallet as a whole